### ISSUE TITLE
bdist_rpm creates RPMs that depend wrongly on vsc-base instead of python-vsc-base.

### ISSUE THREAD

**Issue by piojo** [hpcugent/vsc-mympirun#11]
Same problem as in hpcugent/vsc-ldap#13

**Comment by boegel** [comment]
@stdweird fixed with #52?

**Comment by stdweird** [comment]
looks ok. with current develop, rpm generated with `VSC_RPM_PYTHON=1 python setup.py bdist_rpm`, produces dependencies
```
[stdweird@spikev vsc-mympirun (develop %)]$ rpm -q --requires -p dist/vsc-mympirun-3.4.4-1.noarch.rpm 
/bin/bash
/usr/bin/python
python(abi) = 2.7
python-IPy
python-vsc-base >= 2.5.0
python-vsc-install >= 0.10.9
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PartialHardlinkSets) <= 4.0.4-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
```
